pkg/agent/datamodel/types.go

```diff
@@ -471,8 +473,8 @@ type WindowsProfile struct {
 	AlwaysPullWindowsPauseImage    *bool                      `json:"alwaysPullWindowsPauseImage,omitempty"`
 	ContainerdWindowsRuntimes      *ContainerdWindowsRuntimes `json:"containerdWindowsRuntimes,omitempty"`
 	WindowsCalicoPackageURL        string                     `json:"windowsCalicoPackageURL,omitempty"`
-	WindowsSecureTlsEnabled        *bool                      `json:"windowsSecureTlsEnabled,omitempty"`
-	WindowsGmsaPackageUrl          string                     `json:"windowsGmsaPackageUrl,omitempty"`
+	WindowsSecureTLSEnabled        *bool                      `json:"windowsSecureTlsEnabled,omitempty"`
```

Comment:
does this affect serialization anywhere to json?

Reply:
it shouldn't since I made sure to only update the variable names rather than the json field mappings

Reply:
added some testing around this to be certain

Reply:
potentially we should have sanity check with @AbelHu since this only touches windows, our testing for linux won't catch this well even if the logic is right